operator/src/test/java/oracle/kubernetes/operator/helpers/AdminPodHelperTest.java

```diff
@@ -340,7 +340,7 @@ public void whenAdminServerHasAdditionalVolumesWithReservedVariables_createAdmin
         .withAdditionalVolume("volume2", "/source-$(DOMAIN_NAME)");
 
     assertThat(
-        getCreatedPod().getSpec().getVolumes(),
+        Objects.requireNonNull(getCreatedPod().getSpec()).getVolumes(),
```

Comment:
What is the advantage of using Objects.requireNonNull() here? Thanks

Reply:
It avoids the superfluous warnings that method such as `getSpec()` are capable of returning null. That wrapper informs the compiler and code checker that _in this particular case_, we know it will not return null.